### DISCUSSION TITLE
Metatranscriptome assembly - rnaSPAdes or metaSPAdes?

### DISCUSSION THREAD

**Discussion by sjaenick** [ablab/spades#754] [Q&A]
Hi there,

to assemble a metatranscriptome dataset (Illumina PE), would you recommend
either rnaSPAdes or metaSPAdes?

Thanks in advance.

**Comment by asl** [comment]
Hello

It depends... What you aiming to assemble there? Transcripts? Viruses?

**Comment by sjaenick** [comment]
(Prokaryotic) transcripts, mainly.
